gradio/components.py

```diff
@@ -1977,6 +1979,8 @@ def __init__(
             raise ValueError(
                 f"Invalid value for parameter `source`: {source}. Please choose from one of: {valid_sources}"
             )
+
+        self.subtitle = self.format_subtitle(subtitle)
```

Comment:
Should be added to the docstring

Reply:
It's actually a private function and should not be called from outside. I will add docstring and change it to private.